lib/internal/fs/promises.js

```diff
@@ -125,6 +125,10 @@ async function writeFileHandle(filehandle, data, options) {
   } while (remaining > 0);
 }
 
+// Note: This is different from kReadFileBufferLength used for non-promisified
+// fs.readFile.
+const kReadFileMaxChunkSize = 16384;
```

Comment:
Out of curiosity, is there context for this?

Reply:
Not really, it was there when I found it so I kept it that way. I just thought it was interesting to note.

Reply:
I guess we'd have to ask @jasnell about it.